packages/react-core/src/components/Drawer/DrawerPanelContent.tsx

```diff
@@ -2,6 +2,38 @@ import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Drawer/drawer';
 import { css } from '@patternfly/react-styles';
 import { DrawerContext } from './Drawer';
+import { formatBreakpointMods } from '../../helpers/util';
+
+export const DrawerModifiers = {
+  25: 'width_25',
+  33: 'width_33',
+  50: 'width_50',
+  66: 'width_66',
+  75: 'width_75',
+  100: 'width_100'
+} as const;
```

Comment:
Don't think you need `as const`? Maybe just make an enum?

Reply:
Enum key cannot be a numerical value. :sob: